doc/cephadm/services/rgw.rst

```diff
@@ -237,6 +264,10 @@ where the properties of this service specification are:
     to match the nodes where RGW is deployed.
 * ``virtual_ip``
     The virtual IP (and network) in CIDR format where the ingress service will be available.
+* ``virtual_ips_list``
+    The virtual IP address in CIDR format where the ingress service will be available.
+    Each virtual IP address will be primary on one node running the ingress service. The number
+    of virtual IP addresses needs to be less than or equal to the number of ingress nodes.
```

Comment:
Grr, I should have also suggested s/needs to/must/

Reply:
Done.